src/acmApi/index.ts

```diff
@@ -0,0 +1,34 @@
+import { useAuth0 } from "@auth0/auth0-react";
+import { authorized } from "./authHelper";
+import { AxiosInstance } from "axios";
+
+interface Result {
+    message: string;
+    result: string;
+}
+
+// TODO: sync with types in portal-backend?
```

Comment:
Did some research on this point. Apart from publishing an npm package (not feasible because of the associated cost), we can put all the type definitions into a separate git repository and access it as a sub-module through all other repositories that depend on the same types.

Reply:
Another solution for this happens to be going with the monorepo option. We combine both the backend & frontend into a single repo. Typescript apparently has built in features to optionally run one or other other with different webpack configurations.

Reply:
Let's leave it for the future and discuss further.